src/server/system_services/system_store.js

```diff
@@ -738,6 +740,7 @@ class SystemStore extends EventEmitter {
         _.each(changes.remove, (list, name) => {
             get_collection(name);
             _.each(list, id => {
+                if (!mongo_utils.is_object_id(id)) throw new Error(`SystemStore: make_changes id is not of type object_id: ${id}`);
```

Comment:
@jackyalbo 
What about other ops besides `changes.remove`?
We have the same thing in `insert`, `update`, `db_delete`.
Shouldn't we have the same handling there?

Reply:
@jeniawhite 
changes.update have it - you can take a look again.
insert will fail without id as it part of schema validation
db_delete I will add